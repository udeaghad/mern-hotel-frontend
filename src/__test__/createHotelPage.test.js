import React from 'react';
import { Provider } from  'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/ConfigureStore'
import CreateHotel from '../components/HotelPage'
import { BrowserRouter } from 'react-router-dom';

describe("To test the Create hotel component", () =>{
  test('To test Home page renders successfully!', () => {    
    const tree = renderer
      .create(
      <BrowserRouter>
        <Provider store={store}>
        <CreateHotel />
      </Provider>
      </BrowserRouter>
      )
      .toJSON;
    expect(tree).toMatchSnapshot()
  })

})