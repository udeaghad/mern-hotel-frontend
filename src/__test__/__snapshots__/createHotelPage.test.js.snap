// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`To test the Create hotel component To test Home page renders successfully! 1`] = `[Function]`;
